back/app/policies/user_policy.rb

```diff
@@ -118,6 +118,7 @@ def permitted_attributes_for_update
 
   private
 
+  # TODO: Use a call to CustomFieldParamsService.new.extract_custom_field_values_from_params! instead, but how?
```

Comment:
`allowed_custom_field_keys` is used in `permitted_attributes_for_update` (above, in this file), which in turn is used in `permitted_attributes_for_update`.

Because it returns a list of attributes, not key/value pairs, I am unsure how we could replace the call to `CustomFieldParamsService.new.custom_field_values_params` with one to `CustomFieldParamsService.new.extract_custom_field_values_from_params!`

Reply:
To make it work, we would have to give up on using `permitted_attributes` (which is nice, but kind of an exception in our code base anyway) in the `UsersController`.
 
https://github.com/CitizenLabDotCo/citizenlab/blob/748c5f5ee804fcf5453160e8baba5d45f45d7f4d/back/app/controllers/web_api/v1/users_controller.rb#L113